modules/features/player/src/main/res/layout/fragment_bookmarks_container.xml

```diff
@@ -0,0 +1,57 @@
+<?xml version="1.0" encoding="utf-8"?>
+<layout xmlns:android="http://schemas.android.com/apk/res/android"
```

Comment:
This needs a scroll view as you can't access the bookmarks if they go off the screen.
<img  width="300" src="https://github.com/Automattic/pocket-casts-android/assets/308331/da7add93-d039-41dc-835b-0dd6eea8ddc6" />

<img  width="500" src="https://github.com/Automattic/pocket-casts-android/assets/308331/a9988aa7-c6f3-4afe-a316-9dcd52ce07d7" />